python/tests/maturin_compat.py

```
@@ -0,0 +1,213 @@
+        "-m",
+        "maturin",
+        "build",
+        "--release",
+        "--out",
+        str(out_dir),
+        "--manifest-path",
+        str(root / "tei-py/Cargo.toml"),
+    ]
+    env = os.environ.copy()
+    env["TEI_PY_BUILD_EXTENSION"] = "1"
+    subprocess.run(command, check=True, cwd=root, env=env)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)